executor/cook/executor.py

```diff
@@ -419,6 +417,10 @@ def shutdown(self, driver):
         logging.info('Mesos requested executor to shutdown!')
         self.stop_signal.set()
 
+    def error(self, driver, message):
```

Comment:
Docstring?

Reply:
This is an overridden method like the others above, e.g. `shutdown`, `killTask`, `launchTask`, etc.